inference-engine/src/gna_plugin/gna_plugin.hpp

```diff
@@ -83,6 +83,7 @@ class GNAPlugin : public InferenceEngine::IInferencePlugin {
      */
     uint32_t rwSegmentSize = 0;
 
+    InferenceEngine::CNNNetwork network;
```

Comment:
Can this change affect memory footprint in runtime?

Reply:
fixed, used InputDesc and OutputDesc structures to save the original precisions.